PyInstaller/utils/hooks/gi.py

```diff
@@ -105,33 +106,32 @@ def collect_typelib_data(self):
         binaries = []
         hiddenimports = []
 
-        logger.debug("Collecting module data for %s %s", self.name, self.version)
+        logger.debug(f"Collecting module data for {self.name} {self.version}")
```

Comment:
Actually, `logger` functions have lazy string evaluation, so using printf-style string arguments is preferable to f-strings.

Reply:
Thanks, fixed! :+1: